src/content/views.py

```diff
@@ -11,7 +12,7 @@
 
 
 def index(request):
-    all_news = News.objects.select_related('rating')
+    all_news = News.objects.select_related('rating').annotate(cases=Count('case'))
```

Comment:
Changing `cases` to `cases_count` makes it easier to guess what the property is about. `cases` is too generic and can be mistaken for something similar to `case_set`.

Reply:
Sure it was quite ambiguous, now it's fixed!